### DISCUSSION TITLE
Attachment not visible by the whistleblower once submitted

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3704] [Q&A] [answered]
Hello, why does the whistleblower cannot see its own attachments after submitting the report (while returning to it via 16-digits code)?

![attachm_wb](https://github.com/globaleaks/GlobaLeaks/assets/35804199/bd2083e8-f1ad-4224-a98d-55035cccd555)

The recipient can see (and can download) it correctly.

![attachm_rc](https://github.com/globaleaks/GlobaLeaks/assets/35804199/ee6283a4-0fdc-4e32-abcb-6a044c635d19)

**Comment by evilaliv3** [comment, marked as answer]
@maelaborazioni : Quando globaleaks è stato progettato inizialmente, per ragioni di sicurezza si è pensato di vietare ai segnalanti la possibilità di scaricare i contenuti per limitare il danno nel caso in cui le credenziali fossero andate perse o intercettate.

Nelle prossime versioni stiamo valutando di rivedere questa implementazione permettendo ai segnalanti di caricare i file

**Reply by maelaborazioni** [reply]
Capisco. Se parliamo di foto, video o documenti è presumibile che l'utente mantenga una copia del file sul suo dispositivo.
Mi è sorto invece qualche dubbio nel caso in cui il segnalante decida di lasciare un messaggio vocale in cui descrive ad esempio, i dettagli della segnalazione. Nei casi in cui la segnalazione sia solo vocale (attualmente è uno scenario che potrebbe sostituire la telefonata o il colloquio orale), il segnalante che rientra non ha nessuna informazione circa il contenuto della sua segnalazione.
Spero si sia capito 😅
